config/jest.setup.js

```diff
@@ -53,9 +53,12 @@ ManageIQ.redux.store.injectReducers();
 
 Object.defineProperty(Array.prototype, 'flat', {
     value: function(depth = 1) {
-      return this.reduce(function (flat, toFlatten) {
-        return flat.concat((Array.isArray(toFlatten) && (depth>1)) ? toFlatten.flat(depth-1) : toFlatten);
-      }, []);
+      if (this.reduce) {
+        return this.reduce(function (flat, toFlatten) {
+          return flat.concat((Array.isArray(toFlatten) && (depth>1)) ? toFlatten.flat(depth-1) : toFlatten);
+        }, []);
+      }
+      return this;
```

Comment:
This seemingly just tries to bypass reduce functionality.  I would expect that wherever this is used would probably break?  If it's not used, then we should just remove it entirely

Reply:
This will only affect the jest specs, it doesn't actually break any UI functionality. Adding this if check allows the jest specs to pass.